blog/am2-for-neon.adoc

```diff
@@ -0,0 +1,47 @@
+= JBoss Tools AM2 for Eclipse Neon
+:page-layout: blog
+:page-author: jeffmaury
+:page-tags: [release, jbosstools, jbosscentral]
+:page-date: 2016-07-29
+
+link:/downloads/jbosstools/neon/4.4.1.AM2.html[JBoss Tools 4.4.1 AM2] for Eclipse Neon are here waiting for you. Check it out!
```

Comment:
I would mention that this is a development milestone release. AM2 is not obvious. Especially because we never called our builds AMxyz before. The rest looks good!